scripts/locales/en.lua

```diff
@@ -126,6 +126,7 @@ local lang = {
     ["client_count"] = "Clients Contacts",
     ["client_fingerprint"] = "Client Fingerprint",
     ["client_fingerprint_type"] = "Type",
+    ["client_hash.hassh"] = "HASSH Client Fingerprint",
```

Comment:
Dot (.) is used to identify subtrees in i18n, is it possible to avoid using it in keys?